client.go

```diff
@@ -141,6 +141,10 @@ type Client interface {
 	// with the given subject and name already exists.
 	CreateStream(ctx context.Context, subject, name string, opts ...StreamOption) error
 
+	// DeleteStream deletes a stream. Subject is the NATS subject the stream is
+	// attached to, and name is the stream identifier, unique per subject.
```

Comment:
Docstring is referencing subject which is not an argument on the function.

Reply:
Should be fixed in 7aa8cf1.